### PR TITLE
Update to latest standards

### PR DESCRIPTION
**Pull request by RodEsp**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -20,6 +20,7 @@ module.exports = {
     indent: ['error', 2, {SwitchCase: 0, MemberExpression: 0}],
     quotes: ['error', 'single', {avoidEscape: true}],
     semi: ['error', 'never'],
+    'unicorn/prevent-abbreviations': 'off',
```
Comment (RasPhilCo) [inline index.js:23]: hehe, feel free to remove this i can just start typing `context` lol

Comment (RodEsp) [reply index.js:23]: Haha it was actually messing up a bunch of stuff in `oclif/oclif`, like trying to correct `sourceDir` to `sourceDirection`.

Comment (RasPhilCo) [reply index.js:23]: I just noticed! it was changing type names!

**Review by RasPhilCo — APPROVED** [review]
🏄

**Pull request merged**